build/commands/lib/config.js

```diff
@@ -557,6 +557,12 @@ Config.prototype.update = function (options) {
     this.use_goma = false
   }
 
+  if (options.auto_gn_gen) {
+    this.auto_gn_gen = true;
```

Comment:
It would be great to have this logic as a default one, but to don't break anything, I would like to try it on some CI build for some time, and then if everything looks fine, I want to enable this flag with a simple PR, so it can be reverted easily if anything pops up.